src/nycdb/sql/pluto_latest.sql

```diff
@@ -3,5 +3,20 @@ ALTER TABLE pluto_latest RENAME COLUMN communityboard TO cd;
 ALTER TABLE pluto_latest RENAME COLUMN censustract2010 TO ct2010;
 ALTER TABLE pluto_latest RENAME COLUMN councildistrict TO council;
 ALTER TABLE pluto_latest RENAME COLUMN postcode TO zipcode;
-
+ALTER TABLE pluto_latest ADD COLUMN landusdesc text;
```

Comment:
Just a typo in the column name here

```suggestion
ALTER TABLE pluto_latest ADD COLUMN landusedesc text;
```

Reply:
thank you!!